libs/barista-components/stacked-series-chart/README.md

```diff
@@ -40,21 +40,22 @@ follow the same order given by the developer
-| `visibleValueAxis`       | `boolean`                                                | true     | Visibility of value axis.                                                                                                                                                                                             |
-| `maxTrackSize`           | `number`                                                 | 16       | Maximum size of the track.                                                                                                                                                                                            |
+| Name                     | Type                                                     | Default    | Description                                                                                                                                                                                                           |
+| ------------------------ | -------------------------------------------------------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `mode`                   | `DtStackedSeriesChartMode`                               | `'bar'`    | Display mode.                                                                                                                                                                                                         |
+| `series`                 | `DtStackedSeriesChartSeries[]`                           | -          | Array of series with their nodes.                                                                                                                                                                                     |
+| `selectable`             | `boolean`                                                | false      | Allow selections to be made on chart                                                                                                                                                                                  |
+| `selected`               | `[DtStackedSeriesChartSeries, DtStackedSeriesChartNode]` | -          | Current selection [series, node]                                                                                                                                                                                      |
+| `selectionMode`          | `DtStackedSeriesChartSelectionMode`                      | 'node'     | Whether to make just the nodes selectable or the whole stack                                                                                                                                                          |
```

Comment:
I think it would be nice to add the possible options in the doc: `node`, `stack`